### PR TITLE
new chapter_intro slides

### PR DESCRIPTION
**Pull request by gustavekenedi**
Hi Florian,

Here are our proposed slides for chapter_intro (I've renamed it to make it more explicit than chapter_1).
Because of large file issues I've had to zip "chapter_intro_files/figure-html/contours_iris_2015.zip" so the code for the map won't work unless you unzip that file. The spirit of the lecture has very much stayed the same, but there are some content changes nonetheless.

Feel free to disagree with any of them of course. If you think these slides are likely to take more than 2 hours to cover then I think it would be good to stop at slide 29.
Let us know what you think :)

Thanks!

I'll be making a PR for chapter_summarizing_data quickly.

Gustave

### CODE REVIEW

**Comment by floswald** [comment]
thanks, i'll have a look! 

it's **not** a good idea to rename files. this way I cannot easily see changes you make. what you do when renaming a file is deleting `chapter1.Rmd` and creating a new file `chapter_intro.Rmd`. There is no diff between the two. I'd rather see each line you changed in `chapter1.Rmd`. makes sense? that's for future PRs! 

(we can change filenames as the very last step, if you have strong feelings between `chapter1.Rmd` and `chapter_intro.Rmd`).

**Comment by gustavekenedi** [comment]
I see, very sorry! I can create a new PR without the name change if you'd like!

**Comment by floswald** [comment]
No big deal, if it’s too difficult it’s fine. For next time keep in mind!

**Comment by gustavekenedi** [comment]
I've renamed the folder and files and it seems to work. Will definitely keep it in mind!

**Comment by floswald** [comment]
more things for next time:

* don't commit .html files. those are output
* don't commit anything other than source code (i.e. Rmd or the css if you change it) and stuff i need to compile the html (like new pictures etc)

File: chapter1/chapter1.Rmd [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
 title: "ScPoEconometrics"
 subtitle: "Introduction"
-author: "Florian Oswald"
-date: "SciencesPo Paris </br> `r Sys.Date()`"
+author: "Gustave Kenedi"
```
Comment (floswald) [inline chapter1/chapter1.Rmd]: "Florian Oswald, Gustave Kenedi and Pierre Villedieu" seems more appropriate.

File: chapter1/chapter1.Rmd [hunk 2]
```diff
@@ -363,136 +323,254 @@ paris_sh <- readRDS("paris.RDS")
...
 ```{r, eval = FALSE}
 library(sf)
-plot(paris_sh[,"n"],
-     main = "Number or IRIS by Arrondissement",
-     key.pos = 3, 
-     axes = FALSE, 
-     key.width = lcm(1.3), 
-     key.length = 1.0)
+library(tmap)
+iris_shfl <- read_sf("chapter_intro_files/figure-html/contours_iris_2015/CONTOURS-IRIS.shp") %>%
```
Comment (floswald) [inline chapter1/chapter1.Rmd]: broken path. you should at a minimum click "knit" on your computer before you send of the PR after you made changes.

File: chapter1/chapter1.Rmd [hunk 3]
```diff
@@ -363,136 +323,254 @@ paris_sh <- readRDS("paris.RDS")
...
+  mutate(dep = substr(INSEE_COM,1,2)) %>%
+  select(CODE_IRIS, dep, geometry) %>%
+  filter(dep == "75")
```
Comment (floswald) [inline chapter1/chapter1.Rmd:330]: why don't you just create a new shapefile from this subset? that could then easily be included in ../rds

File: chapter1/chapter1.Rmd [hunk 4]
```diff
@@ -363,136 +323,254 @@ paris_sh <- readRDS("paris.RDS")
...
-```
+* Can be improved but you get this with only __13 lines of code!__
 
+```{r, echo = FALSE}
+library(sf)
+library(tmap)
+iris_shfl <- read_sf("chapter_intro_files/figure-html/contours_iris_2015/CONTOURS-IRIS.shp") %>%
+  mutate(dep = substr(INSEE_COM,1,2)) %>%
+  select(CODE_IRIS, dep, geometry) %>%
+  filter(dep == "75")
```
Comment (floswald) [inline chapter1/chapter1.Rmd]: load again paris subset shapefile from ../rds/

**Review by floswald — CHANGES_REQUESTED** [review]
ok great! there is an issue with a wrong hard coded path, so i can't build this. can you change?

File: chapter1/chapter1.Rmd [hunk 5]
```diff
@@ -348,10 +348,7 @@ tm_shape(iris_map) +
 ```{r, echo = FALSE}
 library(sf)
 library(tmap)
-iris_shfl <- read_sf("chapter_intro_files/figure-html/contours_iris_2015/CONTOURS-IRIS.shp") %>%
-  mutate(dep = substr(INSEE_COM,1,2)) %>%
-  select(CODE_IRIS, dep, geometry) %>%
-  filter(dep == "75")
+iris_shfl <- read_sf("../rds/iris_shapefile_2015.shp")
 iris_income <- readRDS("../rds/iris_inc.rds") %>%
```
Comment (floswald) [inline chapter1/chapter1.Rmd:352]: `iris_inc.rds` is not there.

**Review by floswald — CHANGES_REQUESTED** [review]
great, almost there. the iris_inc file is missing.

File: chapter1/chapter1.Rmd [hunk 6]
```diff
@@ -363,136 +323,251 @@ paris_sh <- readRDS("paris.RDS")
...
 
-# Spatial Plotting with ggplot
+# In Practice: Data Wrangling
 
+--
 
-.pull-left[
-<br>
-<br>
+* Most of the time you will be cleaning data
```
Comment (floswald) [inline chapter1/chapter1.Rmd]: You will spend a lot of time preparing data for further analysis.

File: chapter1/chapter1.Rmd [hunk 7]
```diff
@@ -363,136 +323,251 @@ paris_sh <- readRDS("paris.RDS")
...
+gapminder_dplyr <- gapminder %>% 
+  group_by(continent, year) %>% 
+  summarise(
+    count = n(),
+    mean_lifeexp = mean(lifeExp, na.rm = TRUE),
+    mean_gdppercap = mean(gdpPercap, na.rm = TRUE)
+  )
+```
+
+```{r}
```
Comment (floswald) [inline chapter1/chapter1.Rmd:434]: not visible on slide

File: chapter1/chapter1.Rmd [hunk 8]
```diff
@@ -363,136 +323,251 @@ paris_sh <- readRDS("paris.RDS")
...
+
+```{r, example: gganimate, include = F, cache = T}
+# The package for animating ggplot2
+library(gganimate)
+# As before
+# gg <- ggplot(
+#   data = gapminder %>% filter(continent != "Oceania"),
+#   aes(gdpPercap, lifeExp, size = pop, color = country)
+# ) +
+# geom_point(alpha = 0.7, show.legend = FALSE) +
```
Comment (floswald) [inline chapter1/chapter1.Rmd:489]: i guess you want to show that?

File: chapter1/chapter1.Rmd [hunk 9]
```diff
@@ -363,136 +323,251 @@ paris_sh <- readRDS("paris.RDS")
...
 * *script*: a list of commands collected in a text file, each separated by a new line, to be run one after the other.
 
+* To run a script, you need to highlight the relevant code lines and hit `Ctrl`+`Enter` (Windows) or `Cmd`+`Enter` (Mac).
+
+---
+
+# `RStudio` Layout
+
+```{r, echo = F, out.width = "600px"}
+knitr::include_graphics("chapter_intro_files/figure-html/rstudio.png")
```
Comment (floswald) [inline chapter1/chapter1.Rmd]: wrong path

File: chapter1/chapter1.Rmd [hunk 10]
```diff
@@ -619,27 +695,10 @@ background-image: url(https://media.giphy.com/media/oGzFZek2lszlK/giphy.gif)
...
-
-- Categorical or `factor`
-
-* [You should read more right here!](https://scpoecon.github.io/ScPoEconometrics/R-intro.html#data-types)
-
----
-
-# Vectors
-
+# Vectors => Comment: if you think this lecture up until now lasts more than 2 hours, I would be in favor of moving the rest of this lecture to Lecture 2/3.
```
Comment (floswald) [inline chapter1/chapter1.Rmd]: i think we got to around here, maybe slightly further. why split the presentation though? it's not more *summary statistics* (topic of lecture 2) than the rest.

File: chapter1/chapter1.Rmd [hunk 11]
```diff
@@ -650,436 +709,139 @@ background-image: url(https://media.giphy.com/media/oGzFZek2lszlK/giphy.gif)
...
-.pull-right[
-* We can iterate over things other than numbers:
+1. What is the average value of `total` for state's in the "South", i.e. with `region == "South"`?
+
+1. Create a total_percap variable:
     ```{r}
-    for (i in c("mangos","bananas","apples")){
-      print(paste("I love",i))  # the paste function pastes together     strings
-    }
+    murders$total_percap = murders$total / murders$population * 10000
```
Comment (floswald) [inline chapter1/chapter1.Rmd]: consider adding brackets so it's clear what you multiply with 1000 here

**Review by floswald — CHANGES_REQUESTED** [review]
finally talking about content! 👍 looking good alreay. few remaining comments.

File: chapter1/chapter1.Rmd (hunk 10 above)
Comment (floswald) [reply chapter1/chapter1.Rmd]: remove this comment in the heading and we are good to merge.

**Review by floswald — CHANGES_REQUESTED** [review]
final comment to remove in `vectors` then this is ready.

**Comment by gustavekenedi** [comment]
done :)

**Pull request merged**